post.md

```diff
@@ -377,7 +377,7 @@ Would it have been needed if the React authors had leaned into more of
 [JavaScripts built in features for handling asynchrony](https://crank.js.org/)
 rather than
 [being caught up by (hyped?) experimental frontiers](https://overreacted.io/algebraic-effects-for-the-rest-of-us/)?
-Maybe that's why the react team only really want libraries to adopt it?
+Maybe that's why the React team only really want libraries to adopt it?
 Best to hide the magic behind a library call so we don't have to look at it?
 
-Either way - it's here to stay and they do mention the possibility of introducing more primitives to lubricate it's use in the future, so maybe one day all of this will be ours.
+Either way - it's here to stay and they do mention the possibility of introducing more primitives to lubricate its use in the future, so maybe one day all of this will be ours.
```

Comment:
is it? So you've said it's here but they don't want us to use it, it's an effort to work out how to use it, not really talked about the benefits of it, I don't really come away excited by the concept.

Please tell me why I should be hopeful to get to use this more.